### DISCUSSION TITLE
Power Platform CLI in Azure Cloud Shell

### DISCUSSION THREAD

**Discussion by rajyraman** [Azure/CloudShell#451] [Ideas]
I was watching [Azure Friday](https://youtu.be/B3WGacRHcg0?feature=shared) today and came to know about this repo. Now that Cloud Shell can be ephemeral is it possible to include Power Platform CLI in the default image, so that I don't have to dotnet install it?

I currently install Power Platform CLI as a [dotnet tool](https://learn.microsoft.com/en-us/power-platform/developer/howto/install-cli-net-tool?tabs=windows)

**Comment by mbifeld** [comment]
Hi @rajyraman, glad the Azure Friday video brought you here :).  Based on the new [package inclusion guide](https://github.com/Azure/CloudShell/blob/master/docs/package-inclusion-guide.md), the first step is to get the Power Platform CLI in Azure Linux.  See [here](https://github.com/Azure/CloudShell/blob/master/docs/add-package-into-cloudshell.md/#Check-if-package-is-available-in-Azure-Linux) for more details.
